test/test_evaluation/test_train_evaluator.py

```diff
@@ -1325,9 +1325,9 @@ def test_get_splitter_cv_object(self, te_mock):
         D.data['Y_train'] = np.array([0.0, 0.1, 0.2, 0.3, 0.4, 0.5])
         D.info['task'] = REGRESSION
         evaluator = TrainEvaluator()
-        evaluator.resampling_strategy = KFold
-        evaluator.resampling_strategy_args = {'folds': 4, 'shuffle': True,
+        evaluator.resampling_strategy_args = {'n_splits': 4, 'shuffle': True,
                                               'random_state': 5}
+        evaluator.resampling_strategy = KFold(**evaluator.resampling_strategy_args)
```

Comment:
Is the variable `evaluator.resampling_strategy_args` actually used anywhere now?

Reply:
or could these simply be argument to the constructor?

Reply:
I think this will be a lot of arguments for the constructor. For example:

- shuffle or not
- folds
- groups
- number of repeats
- any new thing that fold split supports

This argument gives us flexibility I think. However if you think otherwise I can remove them and make them arguments, but then this will have to be passed to many classes like smbo, automl, estimators and so on.

Reply:
Oh, okay, in general yes. But in these tests it is basically not necessary, right?